article/sections/_6_graph_discovery.tex

```diff
@@ -181,3 +182,32 @@ \subsection{An application of causal discovery}
 Disclosing and marginalizing over structural uncertainty of this kind is rare in choice modelling, as far as we know.
 Lastly, we gain confidence in our results.
 Through automated tests, we ensure our results are not based on untested or implausible causal assumptions.
+
+\subsection{Experimental extensions}
+\label{sec:experimental-extension}
+
+As just demonstrated, causal discovery algorithms take in data and output causal graphs.
+So far, causal discovery algorithms described in the context of observational data, where there is no random assignment to treatment.
+However, this setting has its limitations.
+Indeed, limitations on a variable's observability lead to causal discovery algorithms sometimes returning graphs with undirected edges, representing latent confounding.
+To overcome issues of confounding, there are causal discovery algorithms that make use of experimental data.
+In particular, one should think of three kinds of experiments: planned, unplanned, and natural.
+
+Planned experiments are what we typically think of as experiments.
+These include A/B tests in technology companies, medical trials for drug certification, randomized controlled trials by economists and social scientists, etc.
+Unplanned experiments are instances of true randomization that were not pre-meditated for experimentation.
+For instance, at multiple technology companies, we have found instances of random allocation (e.g., `np.random.choice` in Python) scattered throughout the companies' code bases, outside of formal experiments and seemingly as leftover placeholders.
```

Comment:
I feel this sentence sounds like we are doing something we aren't supposed to aka look into companies code bases. I don't think it's a big deal but I think we should give an example of where the average person can find an example of unplanned experiments.